packages/jest-cli/src/test_scheduler.js

```diff
@@ -96,6 +96,23 @@ export default class TestScheduler {
         });
         return Promise.resolve();
       }
+      if (testResult.leaks) {
+        const message = [
+          'Your test suite is leaking memory! Please ensure all references are cleaned.',
+          '',
+          'There is a number of things that can leak memory:',
+          '  - Async operations that have not finished (e.g. fs.readFile).',
+          '  - Timers not properly mocked (e.g. setInterval, setTimeout).',
+          '  - Missed global listeners (e.g. process.on).',
```

Comment:
How about: "Global event listeners weren't cleaned up (e.g. process.on)"

Reply:
With #4904 that one is no longer necessary 😃